overlord/ifacestate/handlers.go

```diff
@@ -538,8 +558,29 @@ func (m *InterfaceManager) doAutoConnect(task *state.Task, _ *tomb.Tomb) error {
 		return err
 	}
 
-	chg := task.Change()
+	// content iface has special handling for the default-provider
+	// auto-install
```

Comment:
maybe this comment could be clearer:   // wait for auto-install, started by prerequisites code, for the default-providers of content ifaces so we can auto-connect to them